pkg/ccl/changefeedccl/tableset/watcher.go

```diff
@@ -92,10 +122,53 @@ func (d TableDiff) size() int64 {
 	return d.Added.size() + d.Dropped.size() + int64(d.AsOf.Size())
 }
 
+func (d TableDiff) String() string {
+	return fmt.Sprintf("TableDiff{Added: %s, Dropped: %s, AsOf: %s}", d.Added.String(), d.Dropped.String(), d.AsOf.String())
```

Comment:
nit: only one of `Added`/`Dropped` is ever set right? would be nice if the string representation didn't include the extra field